docs/content/learn/tutorials/metadata/_index.md

```diff
@@ -0,0 +1,783 @@
+
+{{% /tab %}}
+{{% tab %}}
+{{< highlight json "linenos=table" >}}
+{
+  "catalog": {
+    "uuid": "c3da6d1d-c20c-4c7c-ae73-4010167a186"
+  }
+}
+{{< /highlight >}}
+
+Although technically outside of the Metadata Section, the `uuid` of a document is always given as a field of the root element, and shares its structure across all OSCAL document types. This is provided using the [uuid](https://pages.nist.gov/OSCAL/documentation/schema/datatypes/#uuid) datatype. UUIDs are a globally unique identifier randomly generated at the same time an OSCAL document is created. This provides a stable and unique way to refer to a given instance of an OSCAL document. Many tools provide easy ways to generate version 4 UUIDs. For our example we have generated one using a trivial UUIDv4 generator.
+
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml "linenos=table" >}}
+---
+
+catalog:
+  uuid: c3da6d1d-c20c-4c7c-ae73-4010167a186
+
+{{< /highlight >}}
+
+Although technically outside of the Metadata Section, the `uuid` of a document is always given as a field of the root element, and shares its structure across all OSCAL document types. This is provided using the [uuid](https://pages.nist.gov/OSCAL/documentation/schema/datatypes/#uuid) datatype. UUIDs are a globally unique identifier randomly generated at the same time an OSCAL document is created. This provides a stable and unique way to refer to a given instance of an OSCAL document. Many tools provide easy ways to generate version 4 UUIDs. For our example we have generated one using a trivial UUIDv4 generator.
+
+{{% /tab %}}
+{{% /tabs %}}
+
+
+## Starting Simple with Required Elements
+
+
+{{< tabs XML JSON YAML >}}
+
+
+{{% tab %}}
+
+First, we start with the `<title>` element. This should be populated with a human-readable title that is exactly or closely matches the existing name of the document. 
+
+For our example, this would look like:
+
+{{< highlight xml "linenos=table" >}}
+
+<title> Example OSCAL Catalog </title>
+
+{{< /highlight >}}
+
+
+Next we have two date based elements. This uses the OSCAL DataTime data type as defined here: [date-with-timezone](https://pages.nist.gov/OSCAL/documentation/schema/datatypes/#date-with-timezone). `<published>` gives the DateTime of when the document was published for the first time. Since we are publishing this document for the first time today (1/1/2021), the published date will be today, and the time is the moment (or near to it) that this document was published. `<last-modified>` provides the DateTime of the most recent change to this document. As we have never published this particular document before, the DateTime should be identical to the one given by `<published>`. In our example, this will look like:
```

Comment:
```suggestion
Next we have two date-based elements. This uses the OSCAL DataTime data type as defined here: [date-with-timezone](https://pages.nist.gov/OSCAL/documentation/schema/datatypes/#date-with-timezone). `<published>` gives the DateTime of when the document was published for the first time. Since we are publishing this document for the first time today (1/1/2021), the published date will be today, and the time is the moment (or near to it) that this document was published. `<last-modified>` provides the DateTime of the most recent change to this document. As we have never published this particular document before, the DateTime should be identical to the one given by `<published>`. In our example, this will look like:
```